frontend/tests/unittests/pages/FormStaffPage.spec.ts

```diff
@@ -2,6 +2,10 @@ import { createRouter, createMemoryHistory } from 'vue-router';
 import { describe, it, expect } from "vitest";
 import { mount } from '@vue/test-utils';
 import FormStaffPage from '@/pages/FormStaffPage.vue';
+import ForestClientUserSession from '../../../src/helpers/ForestClientUserSession';
```

Comment:
This specific import should be changed to `@/helpers/ForestClientUserSession` to make it more readable.

Reply:
Removed import, so this is no longer needed